lua/astrocommunity/test/neotest/init.lua

```diff
@@ -17,27 +14,16 @@ return {
         end,
       },
     },
-    opts = function()
-      return {
-        -- your neotest config here
-        adapters = {
-          require "neotest-go",
-          require "neotest-rust",
-          require "neotest-python",
-        },
-      }
-    end,
+    opts = {},
     config = function(_, opts)
```

Comment:
Since we already define the config, do we need the opts table?

Reply:
we do not here actually. I will remove it

Reply:
Removed in commit